pkg/client/pods.go

```diff
@@ -73,6 +74,11 @@ func (c *pods) Delete(name string) error {
 	return c.r.Delete().Namespace(c.ns).Resource("pods").Name(name).Do().Error()
 }
 
+// Same as above but with pod termination grace period specified.
+func (c *pods) DeleteGracefully(name string, gracePeriod uint64) error {
```

Comment:
You may want to add a helper to client that takes an api.DeleteOptions and an output version and returns a map[string]string.